Algolia.Search/Index.cs

```diff
@@ -1680,6 +1680,12 @@ public Task<JObject> SaveRuleAsync(JObject queryRule, RequestOptions requestOpti
 			}
 
 			string objectID = (string)queryRule["objectID"];
+
+			if (objectID == "")

```

Comment:
A better way to do it in C# is `objectID.IsNullOrEmpty`. nicer to read and covers the case of null as well.